README.md

```diff
@@ -101,11 +101,9 @@ or
 $ pnpm add telegraf
 ```
 
-### Bot
+### `Telegraf` class
 
-A Telegraf bot is an object containing an array of middlewares which are composed 
-and executed in a stack-like manner upon request. Is similar to many other middleware systems 
-that you may have encountered such as Express, Koa, Ruby's Rack, Connect.
```

Comment:
@wojpawlik Maybe should be left similarity with others middleware systems? This would help better understand middleware for those who wrote in expressjs etc